lib/config-data.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const merge = require('lodash/merge');
+const getHeadings = require('./get-headings');
+const ISO6391 = require('iso-639-1');
+
```

Comment:
Probably clearer if you define defaults outside of the method:
```javascript
const defaults = {
  language: 'en',
  toc: { minLevel: 2, maxLevel: 6 }
}

function configData(data, options) {
  const settings = merge(defaults, options);
  // ...
```
This allows you to also use this to do validation later, using [`_.inRange`](https://lodash.com/docs/4.16.6#inRange):
```javascript
const validMinLevel = inRange(settings.toc.minLevel, defaults.toc.minLevel, defaults.toc.maxLevel);
const validMaxLevel = inRange(settings.toc.maxLevel, defaults.toc.minLevel, defaults.toc.maxLevel);
const isValidRange = (validMinLevel && validMaxLevel);
```